chain/chain/src/doomslug.rs

```diff
@@ -330,6 +332,11 @@ impl Doomslug {
         }
     }
 
+    #[cfg(feature = "adversarial")]
+    pub fn adv_disable(&mut self) {
+        self.threshold_mode = NoApprovals
```

Comment:
Since the threshold mode is defined in the same file, remove the import above and change this line to

```
self.threshold_mode = DoomslugThresholdMode::NoApprovals
```

Reply:
++